src/js/formprint/index.js

```diff
@@ -0,0 +1,98 @@
+import 'js/base/setup';
+import 'js/i18n';
+
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+import { View, CollectionView } from 'marionette';
+import hbs from 'handlebars-inline-precompile';
+
+import 'sass/provider-core.scss';
+import 'sass/app-root.scss';
+
+import './formprint.scss';
+
+import initPlatform from 'js/utils/platform';
+
+import App from 'js/base/app';
+
+import FormsService from 'js/services/forms';
+
+import 'js/entities-service';
+
+import BootstrapService from 'js/services/bootstrap';
+
+import IframeFormBehavior from 'js/behaviors/iframe-form';
+
+const RootView = CollectionView.extend({
+  viewComparator: false,
+  el: 'body',
+  template: hbs``,
+  initialize({ model }) {
+    this.render();
+
+    this.addChildView(new FormIframeView({ model }));
+  },
+});
+
+const FormIframeView = View.extend({
+  behaviors: [IframeFormBehavior],
+  className: 'form__frame',
+  template: hbs`
+    <div class="app-frame__content flex-region">
+      <div class="form__layout">
+        <div class="form__content">
+          <iframe src="/formapp/"></iframe>
+        </div>
+      </div>
+    </div>
+  `,
+});
+
+const FormPrintApp = App.extend({
+  channelName: 'app',
+
+  initialize() {
+    initPlatform();
+  },
+
+  onBeforeStart({ name }) {
+    new BootstrapService({ name });
+  },
+
+  beforeStart({ patientId, formId }) {
+    return [
+      Radio.request('bootstrap', 'fetch'),
```

Comment:
Ideally we do not have to bootstrap for this app

Reply:
Removing this breaks the page and causes errors in the forms service. I'm not really sure why.

Error occurs here: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/services/forms.js#L42

You mentioned in the Shortcut story that there may need to be a modification to the forms service. Is this where the modification would be?